org.eclipse.lsp4j.jsonrpc/src/test/java/org/eclipse/lsp4j/jsonrpc/test/LauncherTest.java

```diff
@@ -46,14 +46,12 @@ static class Param {
 		public String message;
 	}
 	
-	static interface A {
-		@JsonNotification
-		public void say(Param p);
+	interface A {
+		@JsonNotification void say(Param p);
```

Comment:
Not sure about the reformatting that happened at this and other similar places in this commit. Was it intentional?

Reply:
This is from the commit that removes redundant modifiers. So it is intended. Methods declared on interfaces are always public.

Reply:
(Strange, I have received your comment by email, but GitHub seems tor refuse displaying it here.)

By the reformatting, I meant the annotation being moved from its own line.

Reply:
Let's keep it that way. It's just a nitpick, anyway.